ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/OrganisationPolicy.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.ccd.sdk.type;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.ccd.sdk.api.ComplexType;
+import uk.gov.hmcts.ccd.sdk.api.HasRole;
+
+@Data
+@ComplexType(name = "OrganisationPolicy", generate = false)
+@Builder
+public class OrganisationPolicy<R extends HasRole> {
```

Comment:
Same here as Organisation...Should we add @AllArgsConstructor and @NoArgsConstructor.